components/GameCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { Badge } from "./ui/badge";
+import { Button } from "./ui/button";
+
+export default function GameCard() {
```

Comment:
add ability to push props into this component look at `DetailsResponse` type  to check what fields are available